### DISCUSSION TITLE
Changing the iconSet to svg-material-symbols-outlined produces an error, icon named grade does not exist anymore ?

### DISCUSSION THREAD

**Discussion by yaourt** [quasarframework/quasar#17632] [General - Components / Directives / etc] [answered]
I'm trying to change the iconSet in quasarconfig to `'svg-material-symbols-outlined'` but then it gives me the following when doing a `quasar dev` or `quasar build`:

`node_modules/.pnpm/quasar@2.17.2/node_modules/quasar/icon-set/svg-material-symbols-outlined.js:64:2: ERROR: No matching export in "node_modules/.pnpm/@quasar+extras@1.16.13/node_modules/@quasar/extras/material-symbols-outlined/index.mjs" for import "symOutlinedGrade"`

Is there a mistake on my side ?
or is this an issue with the icon named `grade` in `material-icons` that does not exist anymore in `material-symbols-*` ?

**Comment by yusufkandemir** [comment, marked as answer]
It seems that `grade` was removed upstream: https://github.com/marella/material-symbols/commit/318d20524692340f614dd7b3d4a8633f4a987554#diff-97ae9034f40e831c69ee04cb27b8fbce5944438aa307d4b7749ec013b4c12e7eL1390

I am not sure if it was an intended change on their side. I have no idea about their release mentality. There is no mistake on your side.

/cc @hawkeye64 should we roll it back, report this somewhere, or update our iconsets with an equivalent icon?

**Reply by hawkeye64** [reply]
Indeed, Google has removed or renamed this one and we need to replace it.

**Reply by hawkeye64** [reply]
![image](https://github.com/user-attachments/assets/b1d0fd9a-d4a2-4f81-8da9-7824f96dc987)

**Reply by hawkeye64** [reply]
Need to find the equivalent icon. It's used for `rating`, so maybe`star`.

**Reply by yusufkandemir** [reply]
@hawkeye64 yes, `star` looks exactly the same to me. We can use it as a replacement.

**Comment by yaourt** [comment]
Here it is my findings ...

From mdi this is the _grade_ icon : ![image](https://github.com/user-attachments/assets/23cda874-6bd1-46c6-b462-7003b70a31ec)

From mdsi:
- _star_ icon :  ![image](https://github.com/user-attachments/assets/aaa61a62-b10c-4ca8-ae3c-6fc856afbbeb)
- _star_rate_ : ![image](https://github.com/user-attachments/assets/2ed88838-0cf6-4810-8908-f6e276936e5b)

dunno

**Comment by hawkeye64** [comment]
https://github.com/quasarframework/quasar/issues/17640
